index.html

```diff
@@ -1819,6 +1819,70 @@ <h2>The <dfn>NDEFRecordType</dfn> string</h2>
     The <dfn>NFCWriter</dfn> is an object used for writing data to NFC devices
     such as tags.
   </p>
+  <p>
+    A {{NFCWriter}} object has the following <a data-cite=
+    "ECMASCRIPT#sec-object-internal-methods-and-internal-slots">
+    internal slots</a>:
+  </p>
+  <table class="simple">
+    <thead>
+     <tr>
+      <th>Internal Slot</th>
+      <th>Description (<em>non-normative</em>)</th>
+     </tr>
+    </thead>
+    <tbody data-link-for="NFCPushOptions">
+     <tr>
+      <td><dfn>[[\PushOptions]]</dfn></td>
+      <td>
+        This slot holds the provided {{NFCPushOptions}} value.
+        It is initially `null`.
+      </td>
+     </tr>
+     <tr>
+      <td><dfn>[[\PushMessage]]</dfn></td>
+      <td>
+        This slot holds the {{NDEFMessage}} set up to be written.
+        It is initially unset.
+      </td>
+     </tr>
+     <tr>
+      <td><dfn>[[\PushTimer]]</dfn></td>
+      <td>
+        This slot holds the timer associated with the writer.
+        It is initially unset.
+      </td>
+     </tr>
+     <tr>
+      <td><dfn>[[\PushHandler]]</dfn></td>
+      <td>
+        This slot holds the <a>pending push handler</a> function.
+        It is initially unset.
+      </td>
+     </tr>
+    </tbody>
+  </table>
+  <p>
+    The <dfn>pending push options</dfn> is the value of the
+    <a>[[\PushOptions]]</a> internal slot of the
+    <a>pending push tuple</a>'s writer.
```

Comment:
OK I see what you are doing here, but I find it more clear to always get the writer and then refer to the internal slots directly

Reply:
OK, not a big deal. Which if the following should I better do:
1. copy the the following from under the NFCWriter internal slots to under the settings object's internal slots where the other "pending ..." internal slots are defined

> The pending push options is the value of the [[PushOptions]] internal slot of the pending push tuple's writer.
The pending push message is the value of the [[PushMessage]] internal slot of the pending push tuple's writer.
The pending push timer is the value of the [[PushTimer]] internal slot of the pending push tuple's writer.
The pending push handler is the value of the [[PushHandler]] internal slot of the pending push tuple's writer.

2. Remove these definitions and refer to them as the pending tuple's `|writer|.[[PushMessage]]` - something that you tried to avoid for the settings object's internal slots, so rather made text definitions for them. But this might have been because it's lengthy to refer to the settings object.

Reply:
Yes that was done because they are globals and hard to refer too

Reply:
I infer you'd prefer option 2 from above?